sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -1089,11 +1089,11 @@ public virtual Response<ExtractLinkedEntitiesResult> ExtractEntityLinking(string
         /// <param name="language"></param>
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
-        public virtual async Task<Response<ExtractLinkedEntitiesResultCollection>> ExtractEntityLinkingAsync(IEnumerable<string> inputs, string language = default, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<RecognizeLinkedEntitiesResultCollection>> ExtractEntityLinkingAsync(IEnumerable<string> inputs, string language = default, CancellationToken cancellationToken = default)
```

Comment:
Shouldn't this method (and related) be renamed to `RecognizeLinkedEntitiesAsync`?

Reply:
Yes!  Good catch - thanks!  I'll fix this.